src/types/Story.ts

```diff
@@ -183,12 +183,6 @@ export namespace Story {
         PUBLISHED = 'published',
     }
 
-    /**
-     * @deprecated Please use `Status` instead.
-     * @see Status
-     */
-    export import LifecycleStatus = Status;
```

Comment:
> The build on master and this branch was failing. I guess the way we used to export it doesn't work in recent versions of TypeScript or something around our build pipeline

Hm... Might be a change in the newer TS versions. 

A few thoughts: 

1. There are more of `export import` places in this codebase:
   ![image](https://github.com/user-attachments/assets/5417cea1-b62d-4bea-b425-3a6d317f7f6a)

2. Can you please try to use `export const T = X; export type T = X;` instead? Will it work?

3. If we're going to remove (which I don't mind), it has to be a major release because of the BC break.